docs/source/conf.py

```diff
@@ -32,8 +32,6 @@
 html_show_sphinx = False
 html_favicon = '_static/logo-square.png'
 html_theme_options = {
-    "light_logo": "paraglider-color-lightmode.png",
-    "dark_logo": "paraglider-color-darkmode.png",
```

Comment:
I think this was removed because I think @divega there were too many "paraglider" mentions on the homepage. I think it's nice to have it on the sidebar though because the logo will be there for other pages which aren't the homepage. Perhaps we can remove the "Paraglider documentation" and just keep the logo? This is what I've seen done across projects I know: [Ray](https://docs.ray.io/en/latest/), [SkyPilot](https://skypilot.readthedocs.io/en/latest/docs/index.html), [FireSim](https://docs.fires.im/en/stable/), [SkyPlane](https://skyplane.org/en/latest/).

Thoughts @divega?